sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastHashJoinExec.scala

```diff
@@ -498,6 +547,8 @@ case class BroadcastHashJoinExec(
          |    }
          |  }
          |}
+         |// special case for NullAwareAntiJoin, if anyNull in streamedRow, row should be dropped.
```

Comment:
Wow !. Did you take a look at that tiny code diff. Great job !!

Reply:
done 
update code like 

```
case EqualTo(l, r) if canEvaluate(l, left) && canEvaluate(r, right) => Some((l, r))
case EqualTo(l, r) if canEvaluate(l, right) && canEvaluate(r, left) => Some((r, l))
```